portfolio/src/main/java/com/google/sps/servlets/Event.java

```diff
@@ -1,31 +1,63 @@
-package com.google.sps;
+package com.google.sps.servlets;
 import java.util.*;
 
 public final class Event{
     private final String organizer;
     private final String eventName;
     private final String location;
     private final String description;
-    private final Date date;
+    private final Date start;
+    private final Date end;
+    private final String password;
 
-    public Event(String organizer, String eventName, String location, String description, Date date){
+    public Event(String organizer, String eventName, String location, String description, Date start, Date end){
         this.organizer = organizer;
         this.eventName = eventName;
         this.location = location;
         this.description = description;
-        this.date = date;
+        this.start = start;
+        this.end = end;
+        this.password = generatePassword();
+    }
+
+    private String generatePassword(){
```

Comment:
Hey, nice job with this! I think the Apache language library can make generating random strings pretty straightforward (like in one line), but I'm not sure how difficult it is to pull in that library for use. Your solution is clean enough that I wouldn't worry about switching